TrackletAlgorithm/MemoryTemplateBinnedCM.h

```diff
@@ -39,17 +38,18 @@ class MemoryTemplateBinnedCM{
     kNBxBins = 1<<NBIT_BX,
     kNSlots = 1<<NBIT_BIN,
     kNMemDepth = 1<<NBIT_ADDR,
-    kNBitsRZBinCM = NBIT_BIN-kNbitsphibin,
-    kNPhiBins = 1<<kNbitsphibin
+    kNBitsRZBinCM = NBIT_BIN-kNBitsphibinCM,
+    slots = (1<<(NBIT_BX+1))*8+kNBitsphibinCM,
+    entries8 = 36//8*(NBIT_ADDR-NBIT_BIN)+4
```

Comment:
I'm very confused about this line. Where does 36 come from. If I look at the getEntries8{A,B}Slot method it returns an ap_uint<entries8> (so 36 bits), but it seems to only fill 4 of these bits.

Reply:
Replaced with `kNBits_MemAddrBinned`